### DISCUSSION TITLE
Welcome to kissat Discussions!

### DISCUSSION THREAD

**Discussion by arminbiere** [arminbiere/kissat#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MxaWnag** [comment]
is it simple to make a solver cloning method if i want to clone a new solver from the original one?  What about following the method kissat_init()

**Reply by arminbiere** [reply]
Unfortunately Kissat does not support this (nor any other incremental) usage.
